cpu/samd21/cpu.c

```diff
@@ -94,6 +108,21 @@ static void clk_init(void)
     while (!(SYSCTRL->PCLKSR.reg & SYSCTRL_PCLKSR_OSC8MRDY)) {}
 #endif
 
+#if CLOCK_USE_XOSC32_DFLL || !GEN2_ULP32K || !GEN3_ULP32K
```

Comment:
In a subsequent PR this can be easily replaced by a `CLOCK_USE_XOSC32K` or similar